jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -69,6 +70,47 @@ private static JITHelpers jitHelpers() {
 		return helpers;
 	}
 
+	/*
+	 * Public methods to provide access to expected exception
+	 */
+
+	public static void setExpectedException(Class<? extends Throwable> expectedException) {
+		Thread currentThread = Thread.currentThread();
+		String vmThreadName = currentThread.getName();
+		initExceptionMap();
+		
+		if (expectedException == null) {
+			exceptionMap.remove(vmThreadName);
+		}
+		else {
+			exceptionMap.put(vmThreadName, expectedException);
+		}
+	}
+	public static Class<? extends Throwable> getExpectedException() {
+		Thread currentThread = Thread.currentThread();
+		if(exceptionMap == null) {
+			return null;
+		}
+		return exceptionMap.get(currentThread.getName());
+	}
+
+	/*
+	 * Public methods to instantiate and destroy exceptionMap
+	 */
+
+	public synchronized static void initExceptionMap() {
+		if (exceptionMap == null) {
+			exceptionMap = new java.util.concurrent.ConcurrentHashMap<String, Class<? extends Throwable>>();
+		}
+	}
+
+	public static void destroyExceptionMap() {
```

Comment:
I do not see you using this function at all. In which cases do you expect this method to be called?

Reply:
I think this can be removed, I can't think of a scenario when this would need. I'll update that.